viewer/src/views/threejs/cad/primitives.ts

```diff
@@ -6,20 +6,6 @@ import * as THREE from 'three';
 import { PrimitiveAttributes } from '../../../workers/types/parser.types';
```

Comment:
add some information about mesh.frustumCulled = false?

Reply:
Will do in a separate PR since this was here from before